python/ray/experimental/state.py

```diff
@@ -345,10 +348,22 @@ def task_profiles(self):
         executions of that task. The second element is a list of profiling
         information for tasks where the events have no task ID.
     """
+    if start == None:
```

Comment:
the pythonic way to do this is `if start is None:`, this also occurs in a couple other places